grpc/src/compiler/java_generator.cc

```diff
@@ -356,8 +356,8 @@ static void PrintMethodFields(Printer* p, VARS& vars,
     vars["method_field_name"] = MethodPropertiesFieldName(method.get());
     vars["method_new_field_name"] = MethodPropertiesGetterName(method.get());
     vars["method_method_name"] = MethodPropertiesGetterName(method.get());
-    bool client_streaming = method->ClientStreaming();
-    bool server_streaming = method->ServerStreaming();
+    bool client_streaming = method->ClientStreaming() || method->BidiStreaming();
```

Comment:
this file is copied from the GRPC project.. are these fixing from upstream? if not, worth seeing if it has been fixed upstream and pulling in the latest file?